rclcpp/type_adaptation/accelerated_pipeline/README.md

```diff
@@ -40,16 +51,43 @@ Variations in pipelines:
 1. A single `inc_node` that performs *N* steps inline and outputting on `/composite/image_out`.
 2. A chain of *N* `inc_node` instances chained together, outputting on `/pipeline/image_out` 
 
-The goal is to have the throughput of `/pipeline/image_out` match that of `/composite/image_out`. This can be achieved by enabling type adaptaion. When type adpataion is disabled, the bigger the image size used, the more unnecessary memory copying during message transport. This slows the pipeline down relative to the composite node with *N* steps inline. 
+The goal is to have the throughput of `/pipeline/image_out` match that of `/composite/image_out`. This can be achieved by enabling type adaptation. When type adaptation is disabled, the bigger the image size used, the more unnecessary memory copying during message transport. This slows the pipeline down relative to the composite node with *N* steps inline. 
 
-There are following parameters for this pipeline:
+There are following parameters for the `inc_node`:
 * `type_adaptation_enabled` - When true, `inc_node` subscribes and publishes `type_adaptation::example_type_adapters::ImageContainer` type messages. And when false, `sensor_msgs::msg::Image` type is used for subscription and publisher.
 * `inplace_enabled` - When true, configures inc_node to directly modify the received CUDA buffer rather than always copy it first.
 * `proc_count` - The number of increment operations to perform on an image.
 
+### Launch file parameters
+
+| Parameter            | Type     | Default                  | Description                                                          |
+| -------------------- | -------- | ------------------------ | -------------------------------------------------------------------- |
+| `config`             | `string` | `pipeline`               | Graph configuration (pipeline \| composite)                          |
```

Comment:
\ instead of comma?

Reply:
This is for escape sequence for MD to render pipe char. Should I change the separator to a comma?

Reply:
Pipe instead of comma. You used commas earlier to delineate options.

Reply:
Fixed it. Using pipe.